vector-app/build.gradle

```diff
@@ -0,0 +1,554 @@
+import com.android.build.OutputFile
+
+apply plugin: 'com.android.application'
```

Comment:
`com.android.library` ?
**EDIT** OK, I get the direction of the dependency now.